### ISSUE TITLE
[ubuntu] JFreeChart class is not available when trying to get chart

### ISSUE THREAD

**Issue by jarlah** [freeacs/freeacs#261]
Wonder why. is this something related to my server setup or is the installation actually missing the jar or class?

**Comment by jarlah** [comment]
edit /etc/java-8-openjdk/accessibility.properties
to
```

#
# The following line specifies the assistive technology classes 
# that should be loaded into the Java VM when the AWT is initailized.
# Specify multiple classes by separating them with commas.
# Note: the line below cannot end the file (there must be at
# a minimum a blank line following it).
#
assistive_technologies=
```
that said remove org.GNOME.Accessibility.AtkWrapper, assistive_technologies should be blank.

**Comment by jarlah** [comment]
added wiki page 

https://github.com/freeacs/freeacs/wiki/Charts-not-displaying%3F
